src/components/PostgameModal/component.tsx

```diff
@@ -56,7 +56,12 @@ const PostgameModal: React.FC<PostgameModalProps> = (props) => {
   const resolvedWagers = props.resolvedWagers
     .filter((w) => w.game_id === gameId && w.resolved)
     .map((wager) => (
-      { ...wager, time: wager.updated_at, odds: wager.wdl ? wager.odds : wager.winning_pool_share }
+      {
+        ...wager,
+        time: wager.updated_at,
+        odds: wager.wdl ? wager.odds : wager.winning_pool_share,
+        type: 'wager' as WagerLiteral,
+      }
```

Comment:
Since you do this twice might be worth externalizing into another util function - it'd also help clean up `createFeedWager`

Reply:
Yeah good call. That actually helps a ton